tracing-core/src/dispatcher.rs

```diff
@@ -113,7 +113,7 @@
 //! </div>
 //! <div class="example-wrap" style="display:inline-block">
 //! <pre class="ignore" style="white-space:normal;font:inherit;">
-//! <strong>Note</strong>:the thread-local scoped dispatcher <code>with_default</code>
+//! <strong>Note</strong>:the thread-local scoped dispatcher (<code>with_default</code>)
 //! requires the Rust standard library. <code>no_std</code> users should use
```

Comment:
extra newline here
![Screenshot from 2020-06-30 09-54-00](https://user-images.githubusercontent.com/1993852/86154232-a21fdc80-bab7-11ea-8e20-f6b92ec3f351.png)

Reply:
agh, still? it looked normal on my machine...

Reply:
might be a caching issue?

Reply:
Oh, huh. It's messed up in the netlify deploy preview for me, too:
![Screenshot_20200630_101319](https://user-images.githubusercontent.com/2796466/86156152-5c184800-baba-11ea-9078-bbcdb44b90bc.png)

I definitely tested this locally with `cargo doc --open` and it looked right. I hate HTML!!!!

Reply:
what the hell! i messed with it some more and it looks right locally, and it _still_ doesn't look nice on netlify.

Reply:
oh...it looks like netlify has not built a new deploy preview for my follow-up changes, which is why it looks the same. that's cool!!!